packages/optimizely-sdk/lib/core/custom_attribute_condition_evaluator/index.js

```diff
@@ -84,7 +84,7 @@ function evaluate(condition, userAttributes, logger) {
  */
 function isValueValidForExactConditions(value) {
```

Comment:
```suggestion
 function isValueValidForExactConditions(value) {
```
since this only checks the type.